### ISSUE TITLE
switch to `isolationboundaries` by default

### ISSUE THREAD

**Issue by smarunich** [tetrateio/tetrate-service-bridge-sandbox#175]
we also need to update the overlay below...
```     
istio:
      kubeSpec:
        overlays:
          - apiVersion: install.istio.io/v1alpha1
            kind: IstioOperator
            name: tsb-istiocontrolplane
            patches:
              - path: spec.meshConfig.defaultConfig.holdApplicationUntilProxyStarts
                value: true

```

**Comment by shamusx** [comment]
Should be named general such as `global` boundary as default?  Or did we want to name it by env ie: `prod`?

**Comment by smarunich** [comment]
agree on `global`

**Comment by nacx** [comment]
In general, how should we implement version-specific configurations in this repo? For example, in the eshop ingress I'd like to configure WAF by default, but it's only available in 1.6.
Should we implement some sort of version-specific stuff? Like having one template per-version and pick one or the other based on what's configured in the variables, or should we just optimize for the latest GA? (I'm leaning towards the latter, as this is something mostly for demos.
Actually, something we could do is, whenever a new GA is ready, we could cut a branch here named `1.5.x`, etc, so we have the last point in time that was meant for that version, and just use branching to deploy version-specific stuff. WDYT?

**Comment by smarunich** [comment]
Agree on (2) to optimize the repo to follow the releases (and have branches and tags). There are few points being randomly done at this stage, but probably the way we converge to as you described:

1. We have published `release-1.5.x` branch https://github.com/smarunich/tetrate-service-bridge-sandbox/tree/release-1.5.x r last week with the idea as you described (to mimic monorepo story we have).

2. We also do use the tags which finally do align with the software releases, it was not the plan before, but it appears to be now, so all the tags will replicate the changes under the version: https://github.com/smarunich/tetrate-service-bridge-sandbox/tags, i.e. the issued tag will match with the "tested" version.

p.s. I will work on decoupling gitops to respect the (1) and (2) as well.

**Comment by shamusx** [comment]
isolation boundaries default as of 1.7.x
